watertap/examples/flowsheets/gac/gac_ui.py

```diff
@@ -41,6 +42,14 @@ def export_to_ui():
         requires_idaes_solver=True,
         category=FlowsheetCategory.wastewater,
         build_options={
+            "MaterialFlowBasis": {
+                "name": "FilmTransferCoefficientType",
```

Comment:
```suggestion
                "name": "MaterialFlowBasis",
```
I'm assuming this is lingering from a copy + paste